ui/src/app/shared/search-box/search-box.component.html

```diff
@@ -24,5 +24,6 @@
       <span class="search-label d-inline-block" [ngStyle]="styles?.label">{{ 'SHARED.SEARCH_BOX.LABEL' | translate:{organization: (organization | async)} }}</span>
       <span class="fa fa-search form-control-feedback"></span>
     </ng-template>
+    <div id="searchTipsLink"><a [routerLink] = "['/about']" fragment="searchTips">Search Tips</a></div>
```

Comment:
Could you remove the id and associated css and use bootstrap text alignment? https://getbootstrap.com/docs/4.0/utilities/text/#text-alignment

Also, not consistent for spaces. 
```[routerLink] = "['/about']"```
should be
```[routerLink]="['/about']"```

Reply:
Was also thinking, it may look better if the link was right aligned under the input.